### PR TITLE
upd(topgrade-bin): `7.1.0` -> `8.2.0`

### PR DESCRIPTION
**Pull request by Zahrun**

### CODE REVIEW

File: packages/topgrade-bin/topgrade-bin.pacscript [hunk 1]
```diff
@@ -1,9 +1,10 @@
 name="topgrade-bin"
 pkgname="topgrade"
-version="7.1.0"
+version="8.2.0"
 url="https://github.com/r-darwish/topgrade/releases/download/v${version}/topgrade-v${version}-x86_64-unknown-linux-gnu.tar.gz"
 description="Upgrades everything on your system"
-hash="f2f61d5833030ff545dedd42a65d7d75e510017de5748da3dd6a9bd6fb32ff52"
+hash="54fe60ef70b21b34c50c0d342ec120aff3a9522ef44a9737f42d5700aed7a1c3"
+breaks="${pkgname}-deb ${pkgname}-app ${pkgname}-git"
```
Comment (wizard-28) [inline packages/topgrade-bin/topgrade-bin.pacscript:7]:
```suggestion
breaks="${pkgname} ${pkgname}-deb ${pkgname}-app ${pkgname}-git"
```

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/topgrade-bin/topgrade-bin.pacscript (hunk 1 above)
Comment (Zahrun) [reply packages/topgrade-bin/topgrade-bin.pacscript]: Couldn’t this be automated?

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Add `gives`

File: packages/topgrade-bin/topgrade-bin.pacscript (hunk 1 above)
Comment (D-Brox) [reply packages/topgrade-bin/topgrade-bin.pacscript]: I guess, if we could set up a bot

Comment (Zahrun) [reply packages/topgrade-bin/topgrade-bin.pacscript]: I mean pacstall could always consider the packages that have the same name but ends with -deb -git -app, excluding the current one, to break the package

Comment (wizard-28) [reply packages/topgrade-bin/topgrade-bin.pacscript]: Yeah this line will be automatically appended in `2.0.0`, the boilerplate `-deb`,`-app`,`-git`, `pkgname` (the suffix will depend on the `name`, a `-deb` pacscript won't have `${pkgname}-deb` as breaks) would be automatically set to break the package.

Any additional packages listed in the `breaks` variable would simply be appended to the auto generated break list.

Comment (Zahrun) [reply packages/topgrade-bin/topgrade-bin.pacscript]: Great!

**Review by wizard-28 — APPROVED** [review]

**Pull request merged**